src/forms/forms.css

```diff
@@ -0,0 +1,114 @@
+.omrs-checkbox {
+  display: inline-block;
+  vertical-align: middle;
+  user-select: none;
+  cursor: pointer;
+}
+
+.omrs-checkbox input[type="checkbox"] {
+  opacity: 0;
+  position: absolute;
```

Comment:
I don't think we should set this as absolutely positioned, if we can avoid it. The reason is that absolutely positioned elements require a [positioned ancestor](https://stackoverflow.com/questions/45784777/what-is-a-positioned-ancestor). We should not require the user of the styleguide to provide a positioned ancestor for all inputs.

If absolute positioning is required for the checkbox to look correct, we should consider making the `<div class="omrs-checkbox">` element `position: relative` so that it becomes the positioned ancestor for the `<input>`

Reply:
resolved this `.omrs-checkbox label` is the ancestor element